python/taichi/core/util.py

```diff
@@ -251,17 +251,19 @@ def check_exists(src):
         )
 
 
-def prepare_sandbox(src):
+def prepare_sandbox(src=None):
```

Comment:
I feel like the `sandbox` is a bit abused - it's originally to keep a copy of `libtaichi_core.so`. Could you add some comments here to explain the use cases?